### DISCUSSION TITLE
Change color of select chevron

### DISCUSSION THREAD

**Discussion by silveltman** [tailwindlabs/tailwindcss#11227] [Help]
Is it possible to change the color of the select chevron? And if so, what's the recommended way?

Thanks!

**Comment by wongjn** [comment]
If this in regards to the chevron on `<select>` elements styled by [`@tailwindcss/forms`](https://github.com/tailwindlabs/tailwindcss-forms), then you could override [the `background-image`](https://github.com/tailwindlabs/tailwindcss-forms/blob/73d5b012bb44ddb5449a30fbee910767975022ce/src/index.js#L132-L137) with a different color.

**Reply by silveltman** [reply]
I see, so that means so support for using the tailwind color palettes, like text-red-500, fill-red-500 etc. Correct?
